scripts/west_commands/run_common.py

```diff
@@ -25,6 +26,44 @@
 # Don't change this, or output from argparse won't match up.
 INDENT = ' ' * 2
 
+if log.VERBOSE >= log.VERBOSE_NORMAL:
```

Comment:
I am confused. `core.py` doesn't use `run_common.py`, so why are you adding the log replacement here and not in `core.py`?

Reply:
So that the individual runner classes have a common interface for doing logging (defined here), which users of the runner library (like run_common.py) can also rely on.

Reply:
I'm still unclear. Users of the runner library are west-aware, so why not use `west.log` in those? Regarding the runner classes, they can use the standard Python logging mechanisms as they are today?

Reply:
> Users of the runner library are west-aware,

They are today, but making it possible to use this library without using west is the goal here.

> so why not use west.log in those?

That's exactly what this PR is doing for the current user, which is run_common.py.

> Regarding the runner classes, they can use the standard Python logging mechanisms as they are today?

They can, and that is an improvement: it makes them a nicer and moer generic library that plays well with standard Python.